test/contrib/rails/apps/rails6.rb

```diff
@@ -0,0 +1,18 @@
+require 'contrib/rails/apps/application'
+
+module Rails6
+  class Application < RailsTrace::TestApplication
```

Comment:
This `Application` class is defined as a full-blown constant (as opposed to some kind of stub) because of how we setup Minitest to test Rails yeah? I don't think there's anything else to do here, just wanted to clarify that's the case, which would mean this will go away when we finish migrating to RSpec.

Reply:
This is correct!